### PR TITLE
[Feature Fix] Include how to copy file in file organizer information

### PR DESCRIPTION
**Pull request by brandonPurvis**
## Purpose
The file tree's information panel did not include how to copy files. This PR adds instructions for copying files to this information.
This closes https://github.com/CenterForOpenScience/osf.io/issues/3182.

## Changes
Added instructions for coping files to the file tree's information panel.

### CODE REVIEW

File: website/static/js/fangorn.js [hunk 1]
```diff
@@ -1668,7 +1668,8 @@ var FGToolbar = {
                         m('p', [ m('b', 'Select rows:'), m('span', ' Click on a row (outside the name) to show further actions in the toolbar.')]),
                         m('p', [ m('b', 'Select Multiple Files:'), m('span', ' Use command or shift keys to select multiple files.')]),
                         m('p', [ m('b', 'Open Files:'), m('span', ' Click a file name to go to the file.')]),
-                        m('p', [ m('b', 'Open Files in New Tab:'), m('span',  ' Press Command (or Ctrl in Windows) and  click a file name to open it in a new tab.')]),
```
Comment (lyndsysimon) [inline website/static/js/fangorn.js]: Good catch on the inconsistent capitalization of "Command".

File: website/static/js/fangorn.js [hunk 2]
```diff
@@ -1668,7 +1668,8 @@ var FGToolbar = {
                         m('p', [ m('b', 'Select rows:'), m('span', ' Click on a row (outside the name) to show further actions in the toolbar.')]),
```
Comment (lyndsysimon) [inline website/static/js/fangorn.js]: "Select rows:" should be "Select Rows:" to be consistent.

**Comment by lyndsysimon** [comment]
First pass complete - good to go.

If the capitalization thing gets addressed before it's merged, awesome. If not, feel free to submit it as a separate PR.

File: website/static/js/fangorn.js [hunk 3]
```diff
@@ -1665,10 +1665,11 @@ var FGToolbar = {
                 onclick: function(event){
                     var mithrilContent = m('div', [
                         m('h3.break-word.m-b-lg', 'How to Use the File Browser'),
-                        m('p', [ m('b', 'Select rows:'), m('span', ' Click on a row (outside the name) to show further actions in the toolbar.')]),
+                        m('p', [ m('b', 'Select Rows:'), m('span', ' Click on a row (outside the name) to show further actions in the toolbar.')]),
                         m('p', [ m('b', 'Select Multiple Files:'), m('span', ' Use command or shift keys to select multiple files.')]),
                         m('p', [ m('b', 'Open Files:'), m('span', ' Click a file name to go to the file.')]),
-                        m('p', [ m('b', 'Open Files in New Tab:'), m('span',  ' Press Command (or Ctrl in Windows) and  click a file name to open it in a new tab.')]),
+                        m('p', [ m('b', 'Open Files in New Tab:'), m('span',  ' Press command (or ctrl in Windows) and  click a file name to open it in a new tab.')]),
```
Comment (sloria) [inline website/static/js/fangorn.js]: "Command" and "Ctrl" should be capitalized here.

File: website/static/js/fangorn.js [hunk 4]
```diff
@@ -1665,10 +1665,11 @@ var FGToolbar = {
...
+                        m('p', [ m('b', 'Copy Files:'), m('span', ' Press option (or alt in Windows) while dragging a file to a new folder or component.')])
```
Comment (sloria) [inline website/static/js/fangorn.js]: Similarly, "Option" and "Alt" should be capitalized.

**Comment by sloria** [comment]
Made the casing changes during merge.

**Pull request merged**